src/generic_ui/polymer/copypaste.html

```diff
@@ -43,6 +46,14 @@
       <span hidden?='{{ SharingState.NONE === ui.copyPasteSharingState }}'>
         Share a one time connection
       </span>
+      <span hidden?='{{ ui.copyPasteSharingState !== SharingState.NONE || ui.copyPasteGettingState !== GettingState.NONE }}'>
+        Manual Connection
```

Comment:
That was pretty much just because I felt like there should be some text there and neither of the previous titles worked.